Rubberduck.Deployment/BuildRegistryScript.ps1

```diff
@@ -104,7 +104,7 @@ try
 	[System.Reflection.Assembly]::LoadFrom($builderAssemblyPath);
 
 	# Determine if MIDL is available for building
-	$devPath = Resolve-Path -Path "C:\Program Files*\Microsoft Visual Studio\*\*\Common*\Tools\VsDevCmd.bat";
+	$devPath = Resolve-Path -Path "%ProgramFiles%*\Microsoft Visual Studio\*\*\Common*\Tools\VsDevCmd.bat";
```

Comment:
This isn't working. 

```
PS C:\Users\Ben Clothier> Resolve-Path -Path "%ProgramFiles%*\Microsoft Visual Studio\*\*\Common*\Tools\VsDevCmd.bat"
PS C:\Users\Ben Clothier> Resolve-Path -Path "C:\Program Files*\Microsoft Visual Studio\*\*\Common*\Tools\VsDevCmd.bat"

Path
----
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat
```

I don't think `Resolve-Path` understand the environmental expansion. Therefore, you probably need to do this as a separate command and then concatenate to the string being passed to the `Resolve-Path` cmdlet.

Reply:
Hmm.... it works for me. I wonder if it only works on newer PS versions? I have:

![image](https://user-images.githubusercontent.com/25420409/42910037-f27ddcd8-8add-11e8-8d5e-5629258cf397.png)

Reply:
```
Name                           Value
----                           -----
PSVersion                      5.0.10586.117
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.10586.117
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

Might be the 5.0 vs 5.1. Can we avoid forcing contributors from having to have PS 5.1?

Reply:
Also, AV is no longer compiling using MIDL:
 
```
EXEC : warning : Cannot locate the VsDevCmd.bat to initialize C++ build tools; falling back to tlbexp.exe.... [C:\projects\rubberduck\Rubberduck.Deployment\Rubberduck.Deployment.csproj]
```

https://ci.appveyor.com/project/retailcoder/rubberduck/build/2.2.0.3514#L629

Reply:
Tried to download PS 5.0 to repro, but MS have stopped supplying it. The download link just yields a text file with the following:

    As communictaed during the development (https://blogs.msdn.microsoft.com/powershell/2016/04/06/windows-management-framework-5-0-updates-and-wmf-5-1/) and release of WMF 5.1 (https://blogs.msdn.microsoft.com/powershell/2017/03/28/windows-management-framework-wmf-5-1-now-in-microsoft-update-catalog/), starting June 1, 2017, users with WMF 5.0 must upgrade to WMF 5.1 to receive support.

    Please download WMF 5.1 from https://aka.ms/wmf5download

Does it therefore make sense to standardise on 5.1?

Also, not sure why this PR would cause MIDL compilation to fail..